### PR TITLE
Version bumps

### PR DESCRIPTION
**Pull request by dincho**
All nodes has been rotated manually. This is state back sync (state -> code)

### CODE REVIEW

File: terraform/environments/main.tf [hunk 1]
```diff
@@ -5,7 +5,7 @@ variable "vault_addr" {
 module "aws_deploy-main-ap-southeast-1" {
   source            = "../modules/cloud/aws/deploy"
   env               = "main"
-  bootstrap_version = "stable"
+  bootstrap_version = "v2.0.1"
```
Comment (lucafavatella) [inline terraform/environments/main.tf]: I compared 4539fd4 (current stable branch) and v2.0.1.

v2.0.1 was taken from master, and seems more recent.

ok.

File: terraform/environments/main.tf [hunk 2]
```diff
@@ -21,7 +21,7 @@ module "aws_deploy-main-ap-southeast-1" {
   additional_storage_size = 30
 
   aeternity = {
-    package = "https://releases.ops.aeternity.com/aeternity-2.3.0-ubuntu-x86_64.tar.gz"
+    package = "https://releases.ops.aeternity.com/aeternity-2.4.0-ubuntu-x86_64.tar.gz"
```
Comment (lucafavatella) [inline terraform/environments/main.tf]: Ok.

**Comment by lucafavatella** [comment]
`stable` branch name seems kept only in https://github.com/aeternity/infrastructure/blob/2b3c0e9a0abea65f9fcea4b38e15ae850d4d4384/ansible/async_provision.yml#L8

**Comment by lucafavatella** [comment]
and `stable` branch is used also in testnet deploy https://github.com/aeternity/aeternity/blob/b945713443651b9af1148854ac44a84db216b057/.circleci/config.yml#L800

**Comment by lucafavatella** [comment]
Does https://github.com/aeternity/aeternity/wiki/Release-process#24-hours-before-tagging-the-release need update?
> Ensure that the deployment scripts for deployment of testnet by CI are up-to-date. (These are in the stable (not latest) tag of the related Docker image.)

**Comment by dincho** [comment]
> Does https://github.com/aeternity/aeternity/wiki/Release-process#24-hours-before-tagging-the-release need update?
> 
> > Ensure that the deployment scripts for deployment of testnet by CI are up-to-date. (These are in the stable (not latest) tag of the related Docker image.)

No, we'll still keep and use the `stable` docker tag to reference the latest tagged version of the infrastructure.

**Comment by dincho** [comment]
> `stable` branch name seems kept only in

Ticket to get rid of it has been created https://www.pivotaltracker.com/story/show/165567448

**Pull request merged**